src/Memstate.Docs.GettingStarted/QuickStart/Commands/EarnPoints.cs

```diff
@@ -1,6 +1,5 @@
 ﻿namespace Memstate.Docs.GettingStarted.QuickStart.Commands
 {
-
     public class EarnPoints : Command<LoyaltyDB, Customer>
     {
         public EarnPoints()
```

Comment:
You don't need an empty default constructor

Reply:
I have tested without them, and `Wire` is ok with these, but `Json` fails. It's possible I've made a mistake and does in fact work. I did check a few times, ...but would be helpful if you could double check I'm right?

Reply:
Ah, ok, I see what you mean, ... if we're only using Wire with `EarnPoints` then we don't need the empty constructor. I was concerned that readers new to Memstate would wonder why some random Command has an empty constructor and others didn't. It wouldn't be obvious that the reason for some having them is because those are being serialized by a specific Json Serializer. 

I agree, we can remove it, but feel we run the risk that some users will start having some hard to debug errors when some of their DTOs do and do not have public constructors. It would be good if we could find a way to help users more here.